### PR TITLE
Updated various modules with f-strings.

### PR DESCRIPTION
**Pull request by kovitikus**
#### Brief overview of PR changes/additions
Changed strings to fstrings where appropriate.

#### Motivation for adding to Evennia
Readability and performance.

### CODE REVIEW

**Comment by Griatch** [comment]
@kovitikus Is this PR ready for review or is it a WIP that you want to add to for now?

**Comment by kovitikus** [comment]
You can review it. I got enough of it done for now. There's a lot more to go through later on. I'll pick it back up later as I have free time.

File: evennia/accounts/bots.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def basetype_setup(self):
                      "boot:perm(Admin);msg:false();noidletimeout:true()"
         self.locks.add(lockstring)
         # set the basics of being a bot
-        script_key = "%s" % self.key
+        script_key = f"{self.key}"
```
Comment (Griatch) [inline evennia/accounts/bots.py]: While correctly converted, this should probably just be `str(self.key)`

File: evennia/accounts/manager.py [hunk 2]
```diff
@@ -164,9 +164,9 @@ def search_account(self, ostring, exact=True, typeclass=None):
         if typeclass:
             # we accept both strings and actual typeclasses
             if callable(typeclass):
-                typeclass = "%s.%s" % (typeclass.__module__, typeclass.__name__)
+                typeclass = f"{typeclass.__module__}.{typeclass.__name__}"
             else:
-                typeclass = "%s" % typeclass
+                typeclass = f"{typeclass}"
```
Comment (Griatch) [inline evennia/accounts/manager.py]: Probably better to do `str(typeclass)` here.

File: evennia/accounts/bots.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def basetype_setup(self):
                      "boot:perm(Admin);msg:false();noidletimeout:true()"
         self.locks.add(lockstring)
         # set the basics of being a bot
-        script_key = "%s" % self.key
+        script_key = f"{str(self.key)}"
```
Comment (Griatch) [inline evennia/accounts/bots.py]: Ah. No - there is no need for an f-string here. Just make it `str(self.key)`.

**Comment by kovitikus** [comment]
I've resolved your requested changes. Thanks for the heads up!

File: evennia/accounts/manager.py [hunk 4]
```diff
@@ -164,9 +164,9 @@ def search_account(self, ostring, exact=True, typeclass=None):
         if typeclass:
             # we accept both strings and actual typeclasses
             if callable(typeclass):
-                typeclass = "%s.%s" % (typeclass.__module__, typeclass.__name__)
+                typeclass = f"{typeclass.__module__}.{typeclass.__name__}"
             else:
-                typeclass = "%s" % typeclass
+                typeclass = f"{str(typeclass)}"
```
Comment (Griatch) [inline evennia/accounts/manager.py]: This should just be `str(typeclass`. The f-string serves no purpose here.

File: evennia/accounts/bots.py (hunk 3 above)
Comment (kovitikus) [reply evennia/accounts/bots.py]: Resolved.

File: evennia/accounts/manager.py (hunk 4 above)
Comment (kovitikus) [reply evennia/accounts/manager.py]: Resolved.

**Pull request merged**